processing/src/main/java/org/apache/druid/frame/key/KeyOrder.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.frame.key;
+
+public enum KeyOrder
+{
+  NONE(false),
```

Comment:
instead of `NONE` means "hashed", would it make sense to just call it `HASHED`?

Reply:
Hmm. What I was thinking is not that "none" means "not ordered" (in ref to the enum name KeyOrder). Like, a "none" ordered thing might be used as a hash code, or it might not. I'll clarify this with a comment. If you still think it should be HASHED, let me know so I can think about it more.

Reply:
OK, I thought about it more, so much more that it took a few months 😄

NONE actually does make sense rather than HASHED, since we can do hash partitioning along with sorting. (Meaning that ASCENDING or DESCENDING key order may still be involved in hash partitioning.)

I'll add comments discussing this.